docs/pages/operations-guide.md

```diff
@@ -271,12 +271,21 @@ pods:
       ...
 ```
 
+<<<<<<< HEAD
+This has a few _direct_ effects:
+  * Every pod gets its own IP address and its own array of ports
+  * Pods do not use the ports on the host machine
+  * Pod IP addresses can be resolved with the DNS: `<task_name>.<framework_name>.autoip.dcos.thisdcos.directory`
```

Comment:
Use <service_name> vs. <framework_name>.

(Looks like it needs to be updated through the doc.)